controller/backup_controller.go

```diff
@@ -352,7 +352,7 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 	// Otherwise, the Backup CR is created by the backup volume controller, which means the backup already
 	// exists in the remote backup target before the CR creation.
 	// What the controller needs to do for this case is retrieve the info from the remote backup target.
-	if backup.Status.LastSyncedAt.IsZero() && backup.Spec.SnapshotName != "" {
+	if backup.Status.LastSyncedAt.IsZero() && backup.Spec.SnapshotName != "" && bc.backupNotInFinalState(backup) {
```

Comment:
nit: could you modify the comment below because there is a change for the check statement?

Reply:
updated the comment, thanks